Apeswap-Bots/Large-Balance/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "apeswap-4",
+  "version": "0.0.1",
+  "description": "This bot detects accounts with high amounts of GNANA balance.",
+  "chainIds": [
+    56
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest --detectOpenHandles",
+    "format": "prettier --write \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "forta-agent": "^0.1.3",
+    "lru-cache": "^6.0.0",
+    "web3-utils": "^1.7.3"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/jest-when": "^3.5.0",
+    "@types/lru-cache": "^7.5.0",
+    "@types/nodemon": "^1.19.0",
+    "forta-agent-tools": "^2.0.5",
+    "nodemon": "^2.0.8",
+    "prettier": "^2.6.0",
+    "ts-jest": "^27.0.3",
+    "typescript": "^4.3.4"
+  },
+  "repository": "https://github.com/NethermindEth/Forta-Agents/edit/@apeswap/Large-Balance/Apeswap-Bots/Large-Balance"
```

```suggestion
  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/Apeswap-Bots/Large-Balance"
```

(This will be the final url)